crates/sui-tool/src/commands.rs

```diff
@@ -200,7 +207,25 @@ pub enum ToolCommand {
         db_checkpoint_path: PathBuf,
     },
 
-    #[command(name = "replay")]
+    #[clap(name = "download-db-snapshot")]
+    DownloadDBSnapshot {
+        #[clap(long = "epoch")]
+        epoch: u32,
+        #[clap(long = "network")]
+        network: Network,
+        #[clap(long = "genesis")]
+        genesis: PathBuf,
+        #[clap(long = "path", default_value = "/tmp")]
+        path: PathBuf,
+        #[clap(long = "skip-checkpoints")]
+        skip_checkpoints: bool,
+        #[clap(long = "skip-indexes")]
+        skip_indexes: bool,
+        #[clap(long = "num-parallel-downloads", default_value = "2")]
+        num_parallel_downloads: usize,
```

Comment:
nit: could we add some `///` comments for each of these so that `--help` prints out useful info?

Reply:
sure